src/components/Room/UsernameForm.tsx

```diff
@@ -34,58 +33,59 @@ function UsernameForm({ canBeClosed, setCanBeClosed }: UsernameFormProps) {
     // Make sure the username they gave isn't empty / made of only white spaces
     const trimmedInputtedUsername: string = inputtedUsername.trim();
     if (trimmedInputtedUsername === '') {
+      console.log('cannot have empty username');
       // stop now! and display error. user needs to resubmit. - Or use formik/yup
```

Comment:
gotcha -- so this is something we're adding later but for now just console logging

Reply:
yeah, Rose knows how to use formik/yup so I figured we could go through that together!